spec/factories/listings.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :listing do
+    user { nil }
```

Comment:
leave only user. That will call the user factory and create an user associated to this listing